docs/using-getting-started.asciidoc

```diff
@@ -0,0 +1,191 @@
+Chrome/83.0.4103.106 Safari/537.36"
+-----------
+
+In order to map this event to ECS, the contents of the event is associated with the
+appropriate ECS fields.
+
+[options="header"]
+|=====
+| Field Name | Value
+
+// ===============================================================
+
+| @timestamp
+|  `2020-07-15T20:20:48.000Z`
+
+// ===============================================================
+
+| ecs.version
+| 1.5.0
+
+// ===============================================================
+
+| event.category
+| web
+
+// ===============================================================
+
+| event.category
+| access
+
+// ===============================================================
+
+| event.dataset
+| apache.access
+
+// ==============================================================
+
+| event.kind
+| event
+
+// ==============================================================
+
+| event.module
+| apache
+
+// ==============================================================
+
+| event.outcome
+| success
+
+// ==============================================================
+
+| http.request.method
+| GET
+
+// ==============================================================
+
+| http.response.body.bytes
+| 2571
+
+// ==============================================================
+
+| http.response.status_code
+| 200
+
+// ==============================================================
+
+| http.version
+| 1.1
+
+// ==============================================================
+
+| source.address
+| 10.42.42.42
+
+// ==============================================================
+
+| source.ip
+| 10.42.42.42
+
+// ==============================================================
+
+| url.original
+| `/content`
+
+// ==============================================================
+
+| user_agent.device.name
+| Other
+
+// ==============================================================
+
+| user_agent.name
+| Chrome
+
+// ==============================================================
+
+| user_agent.original
+| `Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/83.0.4103.106 Safari/537.36`
+
+// ==============================================================
+
+| user_agent.os.full
+| Mac OS X 10.15.4
+
+// ==============================================================
+
+| user_agent.os_name
+| Mac OS X
+
+// ==============================================================
+
+| user_agent.os.version
+| 10.15.4
+
+// ==============================================================
+
+| user_agent.os.version
+| 83.0.4103.106
+
+|=====
+
+Some items to note about this mapping example:
+
+* As many ECS fields as possible are mapped from the original event
+* Several `event` fields are populated based on ECS categorization
+* The raw log is preserved in the `event.original` field
+* For the simplicity of this example some field sets, such as `agent.*`
+  and `host.*`, have been omitted.
+
+
+[float]
+==== Simplified Search
```

Comment:
I agree, a short intro to the reasons helps frame the rest of the doc.  Here is how I start an email to encourage use of ECS by new customers:

> The Elastic Common Schema (ECS) is an open source specification developed with support from the Elastic user community. ECS defines a common set of fields to be used with data in Elasticsearch, such as logs and metrics.

> ECS enables and encourages the normalization of event data, so that you can better analyze, visualize, and correlate the information represented in your events.  This drives efficiency in your operations.